watershed-iOS/Watershed/Watershed/WPAddFieldReportViewController.m

```diff
@@ -81,13 +81,14 @@ - (void)saveForm {
 //  if parent is task VC, do task id, otherwise to nil
     NSNumber *taskId;
     NSString *miniSiteId;
+    UIViewController *parent;
     UIViewController *previousViewController = [((UINavigationController *)self.parentViewController).viewControllers objectAtIndex:((UINavigationController *)self.parentViewController).viewControllers.count-2];
```

Comment:
i guess ill fix this since melissa is gone T_T but even if i do make delegates, the delegate property should be weak right?